src/EditorFeatures/CSharpTest/ExtractMethod/SelectionValidatorTests.cs

```diff
@@ -1332,7 +1332,7 @@ static void Main()
         [Fact, Trait(Traits.Feature, Traits.Features.ExtractMethod)]
         public async Task DontCrash()
         {
-            await IterateAllAsync(TestResource.AllInOneCSharpCode);
+            await IterateAllAsync(TestResource.AllInOneCSharpCodeVersion6);
```

Comment:
Note: I'm going to file a bug to track this.  Our previous split source approach didn't properly split our test resources file.  Hence when we updated AllInOneCSharpCode to include 7.0 features we only did it for one copy.  The DontCrash test hence never ran against 7.0 code and when doing so fails.  

@Pilchie as FYI